docs/guide/cleanup.md

```diff
@@ -4,21 +4,23 @@ The JavaScript ecosystem has grown massively over the years, in good and bad way
 
 Much of this is in the form of packages which are redundant, bloated or have since fallen out of active maintenance.
 
-A good example of the current efforts to clean our packages up, is [@43081j's ecosystem-cleanup](https://github.com/43081j/ecosystem-cleanup). This project aims to [drive contributions upstream](https://github.com/43081j/ecosystem-cleanup/issues) to many different open-source projects. The project aims to:
+A good example of the current efforts to clean our packages up, is [es-tooling/ecosystem-cleanup](https://github.com/es-tooling/ecosystem-cleanup). This project aims to drive contributions upstream to popular open-source projects. The project aims to:
 
 - Modernize existing packages
 - Migrate from redundant packages
 - Migrate from inactive packages
 - Migrate to lighter and/or faster alternatives
 
+::: info Jump into action!
```

Comment:
what does the dev site render these triple colons as? 👀

Reply:
There are [info, tip, and warning boxes in VitePress](https://vitepress.dev/guide/markdown#custom-containers). You can check how it looks in the deploy preview here https://deploy-preview-17--shiny-salamander-b16dd2.netlify.app/guide/cleanup.html

Reply:
ah awesome! i totally missed the preview link by the bot too 🤦‍♂️  my bad